x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/querydsl/query/SpatialRelatesQuery.java

```diff
@@ -94,6 +94,8 @@ public ShapeRelation shapeRelation() {
      */
     public abstract class ShapeQueryBuilder implements QueryBuilder {
 
+        private float boost = 0.0f;
```

Comment:
This change is needed to allow boosting this query, as now all non scoring queries are explicitly set to a boost score of 0